lib/mayaUsd/ufe/UsdCodeWrapperHandler.h

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright 2023 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#pragma once
```

Comment:
According to our coding standard we don't use pragma once - instead use an include guard.

Reply:
I rechecked and all other files use pragma once. I think you are confusing with another code standard from another project... we have so many differing standards! :)

Reply:
Actually no its in our coding standard. You are correct that we do have some `pragma once` - those should also be cleaned up.